x-pack/plugins/screenshotting/server/screenshots/get_render_errors.ts

```diff
@@ -36,15 +36,15 @@ export const getRenderErrors = async (
       args: [layout.selectors.renderError, layout.selectors.renderErrorAttribute],
     },
     { context: CONTEXT_GETRENDERERRORS },
-    logger
+    kbnLogger
   );
-  span?.end();
 
-  if (errorsFound?.length) {
-    logger.warn(
-      `Found ${errorsFound.length} error messages. See report object for more information.`
-    );
+  const renderErrors = errorsFound?.length;
+  if (renderErrors) {
+    kbnLogger.warn(`Found ${renderErrors} error messages. See report object for more information.`);
   }
 
+  eventLogger.getRenderErrorsEnd({ renderErrors });
```

Comment:
Shall we also handle a case when "render errors" fails?

Reply:
added in 595f479111e2